tensorboard/plugins/projector/vz_projector/vz-projector.html

```diff
@@ -292,6 +292,9 @@ <h2 id="notification-title"></h2>
       <paper-icon-button id="selectMode" alt="Bounding box selection" toggles icon="image:photo-size-select-small"></paper-icon-button>
       <paper-tooltip for="selectMode" position="bottom" animation-delay="0" fit-to-visible-bounds>Bounding box selection</paper-tooltip>
 
+      <paper-icon-button id="editMode" alt="Edit current selection" toggles icon="image:exposure"></paper-icon-button>
+      <paper-tooltip for="editMode" position="bottom" animation-delay="0" fit-to-visible-bounds>Edit current selection</paper-tooltip>
```

Comment:
"Append to current selection" might be clearer, but I'm open to other suggestions if you find that confusing.

Reply:
Appending/selection is one half of the functionality, with the other half being removal/deselection. "Add/remove in current selection" is closer to the functionality that needs to be conveyed, but perhaps that's too long hence "Edit current selection".

Granted, "Edit" may be too ambiguous/vague, and "Appending to current selection" could work where deselection is just a bonus with the main intent being additional selection.